src/components/JaegerIntegration/JaegerResults/SpanTable.tsx

```diff
@@ -1,243 +1,250 @@
-    this.props.spans.map(span => {
-      const app = span.process.serviceName === 'jaeger-query' ? span.process.serviceName : span.operationName;
-      const linkToApp = this.goApp(app);
-      const linkToInMetrics = this.goApp(app, '?tab=in_metrics');
-      const linkToOutMetrics = this.goApp(app, '?tab=out_metrics');
-      const appDefinition = (
-        <>
-          {span.operationName.split('.')[0] +
-            (span.operationName.split('.')[1] ? '(' + span.operationName.split('.')[1] + ')' : '')}
-          {span.tags.some(isErrorTag) && (
-            <ExclamationCircleIcon color={PfColors.Red200} style={{ marginLeft: '10px' }} />
-          )}
-        </>
-      );
-      let number = rows.push({
+  private renderTCPSummary(span: Span) {
+    const info = extractOpenTracingTCPInfo(span);
+    return (
+      <>
+        {this.renderCommonSummary(span)}
+        {info.topic && (
+          <div>
+            <strong>Topic: </strong>
+            {info.topic}
+          </div>
+        )}
+      </>
+    );
+  }
+
+  private rows = () => {
+    // TODO: mark somehow spans with errors
+    return this.props.spans.map((span, idx) => {
+      const split = span.process.serviceName.split('.');
+      const app = split[0];
+      const ns = split.length > 1 ? split[1] : this.props.namespace;
+      const linkToApp = '/namespaces/' + ns + '/applications/' + app;
+      const workloadNs = getWorkloadFromSpan(span);
+      const linkToWorkload = workloadNs
+        ? '/namespaces/' + workloadNs.namespace + '/workloads/' + workloadNs.workload
+        : undefined;
+      return {
         isOpen: false,
         cells: [
+          { title: <>{formatDuration(span.relativeStartTime)}</> },
           {
-            title: linkToApp ? (
-              <Tooltip content={<>Go to App {span.operationName.split('.')[0]}</>}>
-                <Link to={linkToApp} onClick={() => history.push(linkToApp)}>
-                  {appDefinition}
-                </Link>
-              </Tooltip>
-            ) : (
-              appDefinition
+            title: (
+              <>
+                <strong>Application: </strong>
+                <Link to={linkToApp}>{app}</Link>
+                <br />
+                <strong>Workload: </strong>
+                {(linkToWorkload && <Link to={linkToWorkload}>{workloadNs!.workload}</Link>) || 'unknown'}
+                <br />
+              </>
             )
           },
+          { title: this.renderSummary(span) },
           {
             title: (
-              <Tooltip
-                content={
-                  <>
-                    {span.operationName}({span.process.serviceName})
-                  </>
-                }
-              >
-                <span>{span.operationName.slice(0, 40)}...</span>
-              </Tooltip>
+              <>
+                {formatDuration(span.duration)}
+                <br />
+                ... and more stats ...
+              </>
```

Comment:
I don't understand "...and mroe stats..." I think that this is because is still wip maybe I should wait @lucasponce ^^

Reply:
yes it's just a WIP thing :)